### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/4062

### CODE REVIEW

File: migration/mm-box-step5-map-identities.md [hunk 1]
```diff
@@ -47,13 +47,13 @@ If you have many mappings to edit, you can choose to upload a CSV file containin
 Upload your own users and groups mappings using the M
```
Comment (JohanFreelancer9) [inline migration/mm-box-step5-map-identities.md:47]:
```suggestion
Upload your own users and groups mappings using the M.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@get-itips, minor additional edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
@JoanneHendrickson

**Comment by scanum** [comment]
@JoanneHendrickson 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**